.github/workflows/continuous-integration-unit-php.yml

```
@@ -42,11 +42,9 @@ jobs:
             wp: 'trunk'
             experimental: true
 
-    # Disabled as long as WordPress trunk does not properly support PHP 8
-    #          - php: '8.0'
-    #            wp: 'trunk'
-    #            coverage: false
-    #            experimental: true
+          - php: '8.0'
+            wp: 'trunk'
```

```suggestion
            wp: '5.6'
```

We already test against WP trunk with PHP 7.4, which seems enough.